apps/api/src/app/auth/auth.controller.ts

```diff
@@ -190,4 +190,9 @@ export class AuthController {
 
     return await this.authService.getSignedToken(user, organizationId, member as MemberEntity);
   }
+
+  @Get('/self-hosted')
+  async logMeIn() {
+    return this.systemOrganizationService.getCommunityEditionOrganizationToken();
+  }
```

Comment:
@djabarovgeorge let's add an ENV variable here to disable when running in NOVU_ENTERPRISE

Reply:
this could work out of the box because this Controller is available only in the community edition.